src/analyzer/analyzergain.cpp

```diff
@@ -7,18 +7,14 @@
 #include "analyzer/analyzertrack.h"
 #include "analyzer/constants.h"
 #include "track/track.h"
-#include "util/math.h"
 #include "util/sample.h"
 #include "util/timer.h"
 
 AnalyzerGain::AnalyzerGain(UserSettingsPointer pConfig)
-        : m_rgSettings(pConfig) {
-    m_pReplayGain = new ReplayGain();
+        : m_rgSettings(pConfig), m_pReplayGain(std::make_unique<ReplayGain>()) {
```

Comment:
We usually stack class initialization lists.

Reply:
like in the newest latest commit?

Reply:
Yes. Thank you.